functionnal_tests/cypress/integration/account/content_account_spec.js

```diff
@@ -276,6 +276,12 @@ describe('Account page', () => {
         cy.contains('.account__userpreference__setting__spacename', 'You are not a member of any space yet')
       })
     })
+    describe('Profile link button', () => {
+      it('should redirect to user\'s public profile', () => {
```

Comment:
FYI, the standarjs rules tells to use double quotes for strings containing a simple quote instead of backslashing the simple quote
change not required unless ci tells you to ^^